packages/os/migrator.service

```diff
@@ -1,6 +1,8 @@
 [Unit]
 Description=Bottlerocket data store migrator
+DefaultDependencies=no
```

Comment:
nit on the commit message: You don't have to start with `-`

Reply:
Updated.